data/858/982/97/85898297.geojson

```diff
@@ -32,7 +32,8 @@
         "Gro\u00dfer Biberhaufen"
     ],
     "name:deu_x_variant":[
-        "Gr. Biberhaufen"
+        "Gr. Biberhaufen",
+        "Gro\u00dfer Biberhaufen"
```

Comment:
This variant dup's the preferred name, and should be removed?

Reply:
Added another commit, this should be fixed.